tests/fixtures.py

```diff
@@ -0,0 +1,133 @@
+import pytest
+from ocsci.config import ENV_DATA
+from tests import helpers
+from ocs import defaults
+
+
+@pytest.fixture()
+def create_secret(request):
```

Comment:
looks like this can be valuable as a helper function and after you can call it in this fixture.
WDYT?

Reply:
Agree, done